common/source/docs/common-smart-battery-landingpage.rst

```diff
@@ -0,0 +1,43 @@
+.. _common-smartbattery-landingpage:
+
+===============
+Smart Batteries
+===============
+
+ArduPilot supports two types of "smart batteries" that communicate using `SMBus <https://en.wikipedia.org/wiki/System_Management_Bus>`__:
+
+- the `Rotoye BatMon <https://rotoye.com/>`__
+- the `Solo battery <https://www.amazon.com/Solo-Smart-Battery-Rechargeable-Lithium/dp/B01LWAAMV3>`__
+
+While not yet very common, smart batteries are easier to attach and detach from the vehicle and are capable of providing more information on the state of the battery including capacity, individual cell voltages, temperature, etc.
+
+.. toctree::
+    :maxdepth: 1
+
+    Rotoye BatMon <common-smart-battery-rotoye>
```

Comment:
same comment

Reply:
If so, what are the recommended changes?

Reply:
just remove the rotoye line...

Reply:
Would BatMon be added automatically when there is a release on master ?

Reply:
not automatically, but I will move  Upcoming stuff into the main wiki upon its release in a Beta version as they occur